Deprecated items usage validation (#2113)
demisto/demisto-sdk: CHANGELOG.md

* Updated the default general `fromVersion` value on **format** to `6.5.0`
* Fixed an issue where the **validate** command did not fail when the integration yml file name was not the same as the folder containing it.
* Added an option to have **generate-docs** take a Playbooks folder path as input, and generate docs for all playbooks in it.

> * Added new validation to the **validate** command to fail and list all the file paths of files that are using a deprecated integration command / script / playbook.

Reviewer: ```suggestion
* Added a check to **validate**, preventing usage of deprecated integration commands, scripts or playbooks.
```

* Fixed an issue where YAML files contained incorrect value type for `tests` key when running `format --deprecate`.
* Added a deprecation message to the `tests:` section of yaml files when running `format --deprecate`.
* Added the 'integration-get-indicators' commands to be ignored by the **verify_yml_commands_match_readme** validation, the validation will no longer fail if these commands are not in the readme file.

> * Added ***deprecation*** a validation class that lists all the file paths of files that are using a deprecated integration command / script / playbook.

Reviewer: ```suggestion
* Added **validate** check that list all the file paths of files that are using a deprecated integration command / script / playbook.
```

* Added a deprecation message to the `tests:` section of yaml files when running `format --deprecate`.
* Added use case for **validate** on *wizard* objects - set_playbook is mapped to all integrations.
* Added the 'integration-get-indicators' commands to be ignored by the **verify_yml_commands_match_readme** validation, the validation will no longer fail if these commands are not in the readme file.

> * Added new validation to the **validate** command to fail and list all the file paths of files that are using a deprecated integration command / script / playbook.

Reviewer: Please move to the `unreleased` section
